shared/src/main/scala/LiterateLexer.scala

```diff
@@ -125,12 +131,66 @@ def recHelp(token: Object): String =
     case Word(value)        => value
     case AlreadyCode(value) => value
     case LitComment(value)  => value
+    case Number(value)      => value
+    case Variable(value)    => value
     case LambdaBlock(value) => value.map(recHelp).mkString("λ", " ", "}")
     case ListToken(value)   => value.map(recHelp).mkString("[", "|", "]")
     case value: String      => value
 
 def sbcsify(tokens: List[LiterateToken]): String =
-  tokens.map(sbcsify).mkString("", " ", "")
+  val out = mutable.StringBuilder()
```

Comment:
```suggestion
  val out = StringBuilder()
```
This should work afaik. If that doesn't work, see if `new StringBuilder()` does